consensus-engine/src/overlay/tree_overlay/overlay.rs

```diff
@@ -158,8 +171,7 @@ where
         // });
         // let root_size = self.root_committee().len();
         // let committee_size = root_size + children_size;
-        let committee_size = self.root_committee().len();
-        (committee_size * 2 / 3) + 1
+        apply_threshold(self.root_committee().len(), self.leader_threshold)
```

Comment:
I think it's sensible that the threshold is configurable for both the leader supermajority and the 'standard' supermajority. For the flat overlay I only added that for the leader supermajority as the other one is always 0.
Here, I would suggest we use a single threshold and apply it to both

Reply:
Good point. Thank you!: f1e3524